### ISSUE TITLE
Disable --start-maximized and other parameters?

### ISSUE THREAD

**Issue by roblourens** [benderjs/browser-launcher2#55]
I want to launch Chrome but without the `--start-maximized` flag, and possibly without some of the other flags that are enabled by default. A maximized window browser is kind of annoying on a wide 27" display. It would be great if I could specify exactly the options to be passed to chrome.exe without any others being added.

**Comment by akhan-netfore** [comment]
I have an issue with launching chrome with hardware acceleration and would like to start chrome by passing the param --disable-gpu. Any way to pass run parameters to the browser launch command would be great and much appreciated.

In the documentation I see there's a configPath param but there's no documentation on it

**Comment by roblourens** [comment]
Doesn't seem to be much going on in this project. I suspect we need a browser-launcher3 :)

**Comment by jakub-g** [comment]
Yep it's been more than a year since anything substantial landed, I guess everyone involved has stuff to do that are higher prio, @roblourens feel invited to fork and fix some bugs :)

**Comment by vweevers** [comment]
@alessioalex had some plans for a fork, I don't know if that came to fruition. But like I said to him:

> I would favor a fork that's not tied to any brand or parent 
project. And one that follows the openopensource.org manifesto.

Let's get a bunch of contributors in one place and stop history from repeating itself.

**Comment by alessioalex** [comment]
Yep nothing came to fruition yet, but I'm still fond of the idea. Would love to help out if others are interested as well. We could make a separate organization and call it .. browser-launcher3 or something. Like Vincent replied to an email a while back, https://github.com/james-proxy/james-browser-launcher looks to be the most up to date fork so we can fork that initially.

**Comment by mitchhentges** [comment]
In the meantime before an official `browser-launcher3` appears, I've added a [`skipDefaults` option](https://github.com/james-proxy/james-browser-launcher#launchuri-options-callback) to `james-browser-launcher`. It should exist in `1.2.4`

**Comment by Reinmar** [comment]
Hi guys, we finally know that unfortunately we'll not be able to maintain browser-launcher2 anymore. More details in: https://github.com/benderjs/browser-launcher2/issues/57
